src/hal/getpinmap_catena4617.cpp

```diff
@@ -0,0 +1,103 @@
+/*
```

Comment:
Why do we need this? Is it at all different than the Catena 4612?  I'm using the 4612 right now, and it works flawlessly.

Reply:
I thought of having it as I see separate getpinmap files for Catena 4610, 4611 and 4612. As these are the duplicates, can we create a common file getpinmap_catena461x.cpp?